tests/mocha/utils_test.js

```diff
@@ -533,4 +533,25 @@ suite('Utils', function () {
       assert.equal(Blockly.utils.math.toDegrees(5 * quarter), 360 + 90, '450');
     });
   });
+
+  suite('deepMerge', function () {
```

Comment:
Do these tests include merging child objects and arrays with the same keys? The actual child merging bit seems missing here unless I'm missing something.

Reply:
No, but to be fair we literally never actually try to do that. We only use this function to create a deep copy of registered shortcuts (so we're only merging with `Object.null`) and and to copy one set of (non-nested) properties from one theme object to another.